src/pvr2wmc.h

```diff
@@ -1,25 +1,23 @@
-#pragma once
 /*
-*      Copyright (C) 2011 Pulse-Eight
-*      http://www.pulse-eight.com/
```

Comment:
I don't believe that pulse8 thing was accurate to begin with, I believe the
addon was first created from a skeleton of another addon and then gutted,
so the copyright headers were from the other addon.

I'm not aware of any code in that file that is anything other than what
krustyreturns and other contributors like myself have produced. Perhaps the
socket class may be from somewhere. I was more so responding to a comment
on the old repo that our file headers should be cleaned up and made
consistent
On 31/03/2015 4:28 pm, "Martijn Kaijser" <notifications@github.com> wrote:

> In src/pvr2wmc.h
> <https://github.com/kodi-pvr/pvr.wmc/pull/4#discussion_r27455513>:
>
> >  /*
> > -*      Copyright (C) 2011 Pulse-Eight
> > -*      http://www.pulse-eight.com/
>
> is this allowed to be changed?
> @opdenkamp <https://github.com/opdenkamp> ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kodi-pvr/pvr.wmc/pull/4/files#r27455513>.
>